test/services/TaskListDataServiceSpec.scala

```diff
@@ -62,11 +62,18 @@ class TaskListDataServiceSpec extends AnyWordSpec with Matchers with ScalaFuture
         TaskListSectionItem(CapitalRedemption, TaskStatus.NotStarted, Some("CapitalRedemptionPage")),
         TaskListSectionItem(VoidedISA, TaskStatus.NotStarted, Some("VoidedISAPage"))
       ))),
-    TaskListSection(CharitableDonationsTitle, Some(Seq(
+    TaskListSection(
+      CharitableDonationsTitle, Some(Seq(
       TaskListSectionItem(DonationsUsingGiftAid, TaskStatus.Completed, Some("CYAPage")),
       TaskListSectionItem(GiftsOfLandOrProperty, TaskStatus.Completed, Some("CYAPage")),
       TaskListSectionItem(GiftsOfShares, TaskStatus.Completed, Some("CYAPage")),
       TaskListSectionItem(GiftsToOverseas, TaskStatus.Completed, Some("CYAPage"))
+    ))),
+    TaskListSection(
+      InterestTitle, Some(Seq(
```

Comment:
Hi David, Could you please use below testing scenario as an example. As both the tailoring service and Interest service returning the same response. will always match, we need to make sure what happens when there aer different responses 
`    TaskListSection(
      InsuranceGainsTitle, Some(Seq(
        TaskListSectionItem(LifeInsurance, TaskStatus.InProgress, Some("LifeInsurancePage")),
        TaskListSectionItem(LifeAnnuity, TaskStatus.Completed, Some("CYAPage")),
        TaskListSectionItem(CapitalRedemption, TaskStatus.NotStarted, Some("CapitalRedemptionPage")),
        TaskListSectionItem(VoidedISA, TaskStatus.NotStarted, Some("VoidedISAPage"))
      )))`

Reply:
Hi Manish, I think this maybe a case of me simply removing the `InterestTitle` TaskListSection from here as it has its own response and the `InsuranceGainsTitle` TaskListSection is already present in this response.